### ISSUE TITLE
'GifImageFile' object has no attribute '_getexif'

### ISSUE THREAD

**Issue by shanerowden** [haltakov/simple-photo-gallery#67]
During the build step, 
```
Something went wrong while generating the thumbnails:  'GifImageFile' object has no attribute '_getexif'
```
I looked through `media.py` and tried adding an exception catch for `AttributeError` at occasions where I saw getexif being called or a gif ext being part of a conditional statement. I'm pretty sure that gifs generally don't have the same metadata as jpgs. But I'm not certain if its a problem with the image itself or more fundamentally.

**Comment by haltakov** [comment]
Thanks for reporting this and sorry for my late reply. I didn't get an e-mail that there is a new issue, so I see it just now. 

I think the problem is that GIFs don't have a standardized metadata format and also no EXIF. I will fix this in the next release.

**Comment by haltakov** [comment]
This works now. I will release the fix as part of version 1.1.1 in the next 1-2 days.
